.changeset/real-spies-pretend.md

```diff
@@ -2,27 +2,29 @@
 '@astrojs/tailwind': major
 ---
 
-Let tailwind postcss plugin load its config file itself. This changes the `tailwind.config.js` loading behaviour where Tailwind would load the config file from `process.cwd()` instead of the project `root`. You can configure the integration's `config.path` option to load from a specific path instead.
+Let tailwind postcss plugin load its config file itself. This changes the Tailwind config loading behaviour where it is loaded from `process.cwd()` instead of the project `root`.
+
+If your Tailwind config file is not located in the current working directory, you will need to configure the integration's `configFile` option to load from a specific path:
 
 ```js
+// astro.config.mjs
 import { defineConfig } from 'astro/config';
 import tailwind from '@astrojs/tailwind';
 import { fileURLToPath } from 'url';
 
 export default defineConfig({
 	integrations: [
 		tailwind({
-			config: {
-				path: fileURLToPath(new URL('./tailwind.config.js', import.meta.url)),
-			},
+			configFile: fileURLToPath(new URL('./tailwind.config.cjs', import.meta.url)),
 		}),
 	],
 });
 ```
 
-This change also requires a Tailwind config file to exist in your project as Astro's fallback value is no longer provided. It is set up automatically during `astro add tailwind`, but you can also manually create a `tailwind.config.cjs` file in your project root:
+This change also requires a Tailwind config file to exist in your project as a fallback config is no longer provided. It is set up automatically during `astro add tailwind`, but if it does not exist, you can manually create a `tailwind.config.cjs` file in your project root:
```

Comment:
I love the clarity of "if it does not exist!" .. otherwise, you are endorsing both an automatic and possibly a manual one.  :smile: